linkerd/transport-metrics/src/zone/sensor.rs

```diff
@@ -0,0 +1,24 @@
+use crate::zone::TcpZoneMetrics;
+
+#[derive(Clone, Debug)]
+pub struct ZoneMetricsSensor {
+    pub metrics: TcpZoneMetrics,
```

Comment:
this scope can be narrowed, right?

```suggestion
    pub(super) metrics: TcpZoneMetrics,
```

Reply:
This can't be narrowed because it's in the associated type for the client (which itself has to be public).

Reply:
I recognize that the _type_ cannot be narrowed, but I was suggesting that the _field_ scope be narrowed, since it's only referehced in the client module (right?).